### PR TITLE
feat(ncu-ci)!: require `--certify-safe` flag

### PR DESCRIPTION
**Pull request by aduh95**
Or ensure the PR has received at least one approving review since last time it was pushed.

### CODE REVIEW

File: lib/ci/run_ci.js [hunk 1]
```diff
@@ -75,6 +85,7 @@ export class RunPRJob {
 
     try {
       cli.startSpinner('Starting PR CI job');
+      throw 'NOOO'
```
Comment (anonrig) [inline lib/ci/run_ci.js:88]: Is this intended?

File: lib/ci/run_ci.js [hunk 2]
```diff
@@ -16,13 +17,16 @@ const CI_V8_NAME = CI_TYPES.get(CI_TYPES_KEYS.V8).jobName;
...
+  constructor(cli, request, owner, repo, prid, certifySafe) {
     this.cli = cli;
     this.request = request;
     this.owner = owner;
     this.repo = repo;
     this.prid = prid;
     this.prData = new PRData({ prid, owner, repo }, cli, request);
+    this.certifySafe =
+      certifySafe ||
+      new PRChecker(cli, this.prData, request, {}).checkCommitsAfterReview();
```
Comment (anonrig) [inline lib/ci/run_ci.js:29]: Does this check for approvals from collaborators, and not from random people?

File: lib/ci/run_ci.js (hunk 1 above)
Comment (aduh95) [reply lib/ci/run_ci.js:88]: Nooo it's not 😅

**Review by mcollina — APPROVED** [review]
lgtm

**Review by benjamingr — APPROVED** [review]

**Review by MoLow — APPROVED** [review]

**Review by BridgeAR — APPROVED** [review]
RSLGTM

**Review by MoLow — APPROVED** [review]

**Comment by joyeecheung** [comment]
This is currently a bit too restrictive - it should add an exceptions to collaborators otherwise even a collaborator cannot use the label to run the CI wihout an approving review (or after they've pushed a new commit), see https://github.com/nodejs/node/pull/52692

**Comment by ovflowd** [comment]
@aduh95 do you intend to make a patch based on what @joyeecheung mentioned? 👀

**Pull request merged**